rancher/context.go

```diff
@@ -222,6 +236,8 @@ func (c *Context) open() error {
 		return err
 	}
 
+	bytes_binding, _ := json.Marshal(c.Binding)
```

Comment:
Check for error here. Small thing, but try to stick with standard naming conventions for variable names.